spec/features/issues_spec.rb

```diff
@@ -322,112 +320,6 @@
           include_examples 'deleted item is listed in Trash', :issue
           include_examples 'recover deleted item', :issue
         end
-
-        describe 'add evidence', js: true do
```

Comment:
Where did all of this go?

Reply:
New specs are [here](https://github.com/dradis/dradis-ce/pull/973/files#diff-ccbb31fd05b3f301b632274f365f30c66c50ccf8f425687875bd638821d7b1f6)

And [here](https://github.com/dradis/dradis-ce/pull/973/files#diff-47f3c1aa36f81a81a36d2c830ee2190892338ca03f5318dc7447c26759aa9bb7)]